src/renderers/WebGLRenderer.js

```diff
@@ -1175,6 +1179,12 @@ function WebGLRenderer( parameters ) {
 
 		}
 
+		if ( capabilities.multiview ) {
+
+			multiview.attachRenderTarget( camera );
```

Comment:
Should this method be called `multiview.attachCamera()` instead?

Reply:
Yep that sounds cool, changed